examples/react-native/metro.config.js

```diff
@@ -31,12 +31,14 @@ function getInternalPackages(root, internalPackagesDirectory) {
 
       return readdirSync(internalDirectoryRoot, {
         withFileTypes: true,
-      }).map(({ name }) => {
-        const packagePath = path.resolve(internalDirectoryRoot, name);
-        const packageName = require(`${packagePath}/package.json`).name;
-
-        return { packageName, packagePath };
-      });
+      })
+        .filter(({ name }) => name !== '.DS_Store')
+        .map(({ name }) => {
+          const packagePath = path.resolve(internalDirectoryRoot, name);
+          const packageName = require(`${packagePath}/package.json`).name;
+
+          return { packageName, packagePath };
+        });
```

Comment:
Could we make this logic a bit more robust to check if instead there exists a package.json at the packagePath? We could do a try/catch or filtering if that file exists using fs.access or something similar?

Reply:
Yeah, good point, let me take a look!

Reply:
+1